CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 4.16.0
+
+* `FEAT`: add `Binding` entry to busines rule task, call activity and user task ([bpmn-io/bpmn-js-properties-panel#1067](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/1067))
```

Comment:
Until 4.15.0 we used "Key changes in properties panel" as a separate section for such changes.

Reply:
Done via https://github.com/camunda/camunda-bpmn-js/pull/374/commits/895f2dd8e00ce07e15bacb3355204ef843e7cb7e